src/components/Header.js

```diff
@@ -9,9 +9,9 @@ import logoSvg from '../assets/logotext.svg'
 
 function Header({ compact }) {
   const BeeIcon = (
-    <a href="https://1hive.org">
+    <Link href="/" external={false}>
```

Comment:
Ah sorry, forgot we are using a hash router, lets add the `#`

```suggestion
    <Link href="/#" external={false}>
```